scripts/test.sh

```diff
@@ -0,0 +1,58 @@
+cwd=$(pwd)
+file_checker_root_path="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )/.."
+cd $file_checker_root_path
+
+error_code=0
+
+echo '''
+================================================================================
+                                      Nose
+================================================================================
+'''
+nosetests . -v --with-coverage --cover-erase
+if [ $? -ne 0 ]; then
+    error_code=1
+fi
+
+echo '''
+================================================================================
+                                     Flake8
+================================================================================
+'''
+flake8 . --exclude env
+if [ $? -ne 0 ]; then
+    error_code=1
+fi
+
+echo '''
+================================================================================
+                                   Run tests
+================================================================================
+'''
+python3 tests/test_util.py
+python3 tests/test_file_checker.py
+if [ $? -ne 0 ]; then
+    error_code=1
+fi
+
+echo '''
+================================================================================
+                                   Run main
+================================================================================
+'''
+python3 src/file_checker.py
+if [ $? -ne 0 ]; then
+    error_code=1
+fi
+
+echo '''
+================================================================================
+'''
+if [[ $error_code -eq 0 ]]; then
+    echo 'Passed'
+else
+    echo 'Error'
+    exit 1
```

Comment:
Exit with the variable error code; It's clearer.

Reply:
There has few command will return error code, do you have any ideas to echo those?

I think `exit 1` is fine, since those commands will also show its own errors.

Reply:
@arnavb I have added echo after each command with its error code.